lambdas/bulk-migrate-vcs/src/main/java/uk/gov/di/ipv/core/bulkmigratevcs/BulkMigrateVcsHandler.java

```diff
@@ -418,7 +419,8 @@ private void logError(
 
         LOGGER.error(
                 annotateLog(message, reportItem, batchId, pageSummary)
-                        .with(LOG_ERROR_DESCRIPTION.getFieldName(), error.getMessage()));
+                        .with(LOG_ERROR_DESCRIPTION.getFieldName(), error.getMessage())
+                        .with(LOG_ERROR_STACK.getFieldName(), error.getStackTrace()));
```

Comment:
Does this just work? I don't know how log4j will handle `StackTraceElement[]`, but hopefully it does something reasonable.